docs/UserGuide.adoc

```diff
@@ -67,105 +72,112 @@ Examples:
 
 * `help`
 
-* `man add`
+* `man task`
+
+=== Adding a task: `task`
+
+Adds a task to the board +
+Format: `add n/NAME d/DATETIME [r/REMARK]`
+
+Examples:
 
-=== Adding a person: `add`
+* `task n/Software Engineering Milestone 1 d/16/3/2018 17:00 r/urgent`
+* `task n/Programming Methodology Assignment 2 d/16/3/2018 23:59`
 
-Adds a person to the address book +
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...`
+=== Adding an event: `event`
 
-[TIP]
-A person can have any number of tags (including 0)
+Adds a event to the board +
+Format: `event n/NAME s/STARTTIME e/ENDTTIME [l/LOCATION] [r/REMARK]`
 
 Examples:
 
-* `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`
-* `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal`
+* `event n/Software Project s/1/5/2018 8:00 e/1/8/2018 8:00 l/School of Computing r/remember to bring laptop charger`
+* `event n/Blockchain Talk s/16/3/2018 16:00 e/16/3/2018 18:00`
 
-=== Listing all persons : `list`
+=== Listing all uncompleted tasks: `ls task`
 
-Shows a list of all persons in the address book. +
-Format: `list`
+Shows a list of all uncompleted tasks in the board. +
```

Comment:
I think should standardize put desk board?

Reply:
I am hesitant whether to put desk board or board, because 'desk board' seems a weird term; I think use board.